components/gcp/container/component_sdk/python/kfp_component/google/ml_engine/_wait_job.py

```diff
@@ -18,14 +18,15 @@
 from ._client import MLEngineClient
 from .. import common as gcp_common
 
-def wait_job(project_id, job_id, wait_interval=30):
+def wait_job(project_id, job_id, wait_interval=30, show_tensorboard=True):
```

Comment:
>show_tensorboard=True

Is it set to False anywhere?

Reply:
It will be set to False when we use wait-job for non-training cmle jobs.
            arguments=[
                "kfp_component.google.ml_engine",
                "wait-job",
                "--project_id",
                gcp_project_id,
                "--job_id",
                job_id,
                "--wait_interval",
                wait_interval_in_seconds,
                "--show_tensorboard",
                "False",
            ],